cs3api4lab/config/config_manager.py

```diff
@@ -9,34 +9,23 @@ class Config(LoggingConfigurable):
     config = None
     __config_dir = "\\jupyter-config"
     __config_file_name = 'jupyter_cs3_config'
-    __env_names = {"reva_host",
-                   "auth_token_validity",
-                   "endpoint",
-                   "home_dir",
-                   "chunk_size",
-                   "secure_channel",
-                   "client_cert",
-                   "client_key",
-                   "ca_cert",
-                   "client_id",
-                   "client_secret",
-                   "login_type"}
+    __env_vars_req = {"reva_host",
+                      "client_id",
+                      "client_secret"}
```

Comment:
As I mentioned in #26, the secret won't be necessary in the configuration if we have different authenticators (this is still not merged).
So maybe we should't check it here.